lib/libzfs/libzfs_import.c

```diff
@@ -173,8 +175,23 @@ fix_paths(nvlist_t *nv, name_entry_t *names)
 				break;
 			}
 
-			if (best == NULL || ne->ne_order < best->ne_order)
+			if (best == NULL) {
```

Comment:
The 3 consecutive `if-continue` blocks here at the end of the loop seem a bit awkward since they follow two `if-break` blocks.  Would it be more clear as an `if / else if / else if` without the `continue` since it's at the end of the loop block?  It could also be done as an `if (a || b || c)` since the if-blocks are all the same, but that might be less readable here.  Yeah, I guess I'm nitpicking.